pkg/config/node.go

```diff
@@ -409,10 +409,19 @@ func GetLBConfig(kubeconfigPath string, apiPort, lbPort, statPort uint16, apiVip
 		config.FrontendAddr = "::"
 	}
 	// Try reading master nodes details first from api-vip:kube-apiserver and failover to localhost:kube-apiserver
-	backends, err := getSortedBackends(kubeconfigPath, false)
-	if err != nil {
-		log.Infof("An error occurred while trying to read master nodes details from api-vip:kube-apiserver: %v", err)
-		log.Infof("Trying to read master nodes details from localhost:kube-apiserver")
+	isBootstrap := os.Getenv("IS_BOOTSTRAP") == "yes"
+	var err error
+	var backends []Backend
+	// On the bootstrap we only want to look at the local apiserver. Once that goes
+	// down we want to shut down keepalived so the API VIP can move to the masters.
+	if !isBootstrap {
+		backends, err = getSortedBackends(kubeconfigPath, false)
+	}
+	if err != nil || isBootstrap {
+		if !isBootstrap {
+			log.Infof("An error occurred while trying to read master nodes details from api-vip:kube-apiserver: %v", err)
+			log.Infof("Trying to read master nodes details from localhost:kube-apiserver")
```

Comment:
I'm not sure if we need this change, 
the GetLBConfig function is used also by haproxy-monitor and IS_BOOTSTRAP ENV var isn't set in haproxy-monitor container.
Additionally, the  bootstrap's kubeconfig is pointing to localhost

Reply:
It's been a while since I wrote this, but IS_BOOTSTRAP was being checked before in the monitor (see line 276 in dynkeepalived). If that's not correct we can remove it, but I think it was necessary here to maintain the same behavior as before.

You make a good point that kubeconfig is already pointed at localhost on bootstrap, but I think part of the reason for this logic was to avoid the log message about the api-vip when the api-vip is not actually what we care about. Maybe I should update the comment though?